src/ocm/components/clusterConfiguration/HostInventory.tsx

```diff
@@ -53,15 +53,15 @@ const PlatformIntegrationLabel: React.FC = () => (
           </List>
         </>
       }
-      buttonOuiaId="platform-integration-vSphere-popover"
+      ouiaId="platform-integration-vSphere-popover"
```

Comment:
@jgyselov I'm assuming Pendo won't have a problem clicking on the element as long as it has the same ID. But it won't be a button anymore (it's what was causing the styling problems)